examples/DayPickerRangeControllerWrapper.jsx

```diff
@@ -143,12 +145,13 @@ class DayPickerRangeControllerWrapper extends React.Component {
 
     return (
       <div style={{ height: '100%' }}>
-        {showInputs &&
+        {showInputs
+          && (
```

Comment:
personally i'd usually do this like `{showInputs && (`

Reply:
@ljharb fixed and added more tests :)